commit c214fd6e5e11fc011f02ece457e9a82b260e046d
Author: Florent Collot <florent.collot@ihu-liryc.fr>
Date:   Fri Sep 28 15:04:27 2018 +0200

    [PATCH] dtkLibraries

diff --git a/cmake/FindDTK.cmake.in b/cmake/FindDTK.cmake.in
index a85d2181..3b2cc3ae 100644
--- a/cmake/FindDTK.cmake.in
+++ b/cmake/FindDTK.cmake.in
@@ -29,8 +29,6 @@ set(dtk_CMAKE_DIRS   "@dtk_CMAKE_DIR@")
 
 set(dtk_USE_FILE "@dtk_USE_FILE@")
 
-set(DTK_LIBRARIES dtkMath dtkCore dtkGui dtkComposer)
-
 set(QT_QMAKE_EXECUTABLE "@QT_QMAKE_EXECUTABLE@")
 
 include(${dtk_USE_FILE})
diff --git a/cmake/dtkUse.cmake.in b/cmake/dtkUse.cmake.in
index 98438773..a2a2c356 100644
--- a/cmake/dtkUse.cmake.in
+++ b/cmake/dtkUse.cmake.in
@@ -31,4 +31,10 @@ if(NOT DTK_USE_FILE_INCLUDED)
    include(dtkBuild)
    include(dtkReporting)
 
+   set(DTK_LIBRARIES dtkMath dtkCore dtkGui dtkLog)
+
+   if(@DTK_BUILD_COMPOSER@)
+      set(DTK_LIBRARIES ${DTK_LIBRARIES} dtkComposer)
+   endif(@DTK_BUILD_COMPOSER@)
+
 endif(NOT DTK_USE_FILE_INCLUDED)
