From 8a28c94003af916137b4126b292cbc7aed855087 Mon Sep 17 00:00:00 2001
From: mathildemerle <mathilde.merle@ihu-liryc.fr>
Date: Tue, 6 Nov 2018 11:38:47 +0100
Subject: [PATCH] QtDCM_Bruker_Minc

---
 Code/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Code/CMakeLists.txt b/Code/CMakeLists.txt
index 7a51a72..c4a8227 100644
--- a/Code/CMakeLists.txt
+++ b/Code/CMakeLists.txt
@@ -101,6 +101,7 @@ set(QTDCM_LIBRARIES
   ITKIOImageBase
   ITKCommon
   ITKIOBMP
+  ITKIOBruker
   ITKIOBioRad
   ITKIOHDF5
   ITKIOGDCM
@@ -109,6 +110,7 @@ set(QTDCM_LIBRARIES
   ITKIOJPEG
   ITKIOLSM
   ITKIOMeta
+  ITKIOMINC
   ITKIONIFTI
   ITKIONRRD
   ${ITKIOPhilipsREC_LIBRARIES}
-- 
2.7.4

